cfpb/sbl-frontend
# [Resolve Errors] [Review Warnings] Tables - full borders will only show when overflowing (horizontal scrolling active)

closes #547 

Note: Requires https://github.com/cfpb/design-system-react/pull/358 to be merged in first

## Changes
- enhancement-style(tables): full borders will only show for overflowing
- chore(tables): removed the need for `showTableBorders`

## How to test this PR

1. `yarn install` to get the latest DSR update with the table ref change
2. use [all-errors-warnings-no-syntax-errors.csv](https://github.com/cfpb/sbl-frontend/files/15444623/all-errors-warnings-no-syntax-errors.csv) or any register with multifield errors that causes overflowing/horizontal scrolling
3. Verify that tables without overflowing don't have full borders, tables that do have overflowing/horizontal scrolling have full borders

## Screenshot
<img width="638" alt="Screenshot 2024-05-25 at 11 47 57 AM" src="https://github.com/cfpb/sbl-frontend/assets/13324863/635e1840-4ef4-4e86-8fc9-679f62cc3903">
